source/plugins/go/src/utils.go

```diff
@@ -123,7 +123,7 @@ func ToString(s interface{}) string {
 
 //mdsdSocketClient to write msgp messages
 func CreateMDSDClient(dataType DataType, containerType string) {
-	mdsdfluentSocket := "/var/run/mdsd/default_fluent.socket"
+	mdsdfluentSocket := "/var/run/mdsd-ci/default_fluent.socket"
 	if containerType != "" && strings.Compare(strings.ToLower(containerType), "prometheussidecar") == 0 {
 		mdsdfluentSocket = fmt.Sprintf("/var/run/mdsd-%s/default_fluent.socket", containerType)
```

Comment:
since MDSD_ROLE_PREFIX changed, for the prometheus, it will be "/var/run/mdsd-ci-%s/default_fluent.socket", right? please double check on this.

Reply:
no. the new paths will be /var/run/mdsd-ci/ and /var/run/mdsd-PrometheusSidecar